src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -1080,6 +1080,7 @@ private void JSON_response(HttpServletResponse response, String jsonValue,
 		try {
 			// output json
 			response.setContentType(MediaType.APPLICATION_JSON);
+			response.setCharacterEncoding("UTF-8");
```

Comment:
Is this being set for other responses?  Any reason that any request wouldn't be UTF-8?